test/index.js

```diff
@@ -1030,12 +1030,14 @@ describe('utility functions', () => {
     await patchNextFiles(process.cwd())
     const serverFile = path.resolve(process.cwd(), 'node_modules', 'next', 'dist', 'server', 'base-server.js')
     const patchedData = await readFileSync(serverFile, 'utf8')
-    expect(patchedData.includes('_BYPASS_SSG')).toBeTruthy()
+    expect(patchedData.includes('_REVALIDATE_SSG')).toBeTruthy()
+    expect(patchedData.includes('private: isPreviewMode && cachedData')).toBeTruthy()
 
     await unpatchNextFiles(process.cwd())
 
     const unPatchedData = await readFileSync(serverFile, 'utf8')
-    expect(unPatchedData.includes('_BYPASS_SSG')).toBeFalsy()
+    expect(unPatchedData.includes('_REVALIDATE_SSG')).toBeFalsy()
```

Comment:
Related to my comment above - while I know `_REVALIDATE_SSG` is being explicitly set to `'true'`, the fact that this statement is passing makes me a little wary about unexpected behaviour if  `_REVALIDATE_SSG` is set to `'false'` since `toBeFalsy` checks if the value is any of false, 0, '', null, undefined, and NaN, and everything else is true (from the [Jest docs](https://jestjs.io/docs/expect#tobefalsy))